tools/src/vendoring/config/expoGoConfig.ts

```diff
@@ -4,6 +4,7 @@ import path from 'path';
 
 import { Podspec } from '../../CocoaPods';
 import { EXPO_DIR, EXPOTOOLS_DIR } from '../../Constants';
+import { getExpoRepositoryRootDir, getReactNativeSubmoduleDir } from '../../Directories';
```

Comment:
```suggestion
import { EXPO_DIR, REACT_NATIVE_SUBMODULE_DIR } from '../../Constants';
```

also use the import names in the following code.